src/components/TxData.js

```diff
@@ -116,7 +116,8 @@ class TxData extends React.Component {
     });
 
     try {
-      const fundsData = await this.props.wallet.graphvizNeighborsQuery(
+      const wallet = getGlobalWallet();
+      const fundsData = await wallet.graphvizNeighborsQuery(
```

Comment:
```suggestion
      const { graphvizNeighborsQuery } = getGlobalWallet();
      const fundsData = await graphvizNeighborsQuery(
```

Reply:
It's not possible to use this syntax with a HathorWallet method: it crashes the application.